subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/CompositeBuildArtifactTransformIntegrationTest.groovy

```diff
@@ -77,8 +75,8 @@ class CompositeBuildArtifactTransformIntegrationTest extends AbstractCompositeBu
         assertTaskExecuted(":buildB", ":jar")
         assertTaskExecuted(":buildC", ":jar")
 
-        outputContains("Transformed artifact: buildB-1.0.jar.xform (project :buildB), location: ${expectedWorkspaceLocation(buildB)}")
-        outputContains("Transformed artifact: buildC-1.0.jar.xform (project :buildC), location: ${expectedWorkspaceLocation(buildC)}")
+        outputContains("Transformed artifact: buildB-1.0.jar.xform (project :buildB)")
```

Comment:
❓ Would we need a test here that this works as well for incremental transforms?

Reply:
I don't think we need that here. This is not about testing _how_ the transform is executed, but about the fact that you can use the output of an included build as an input to (some) transform.

Reply:
I am not sure if it makes a difference whether the transform is incremental or not for the reuse. I'd be happier if there were a test for this.